sapi/cli/php_cli.c

```diff
@@ -574,7 +574,7 @@ static zend_result cli_seek_file_begin(zend_file_handle *file_handle, char *scri
 {
 	FILE *fp = VCWD_FOPEN(script_file, "rb");
 	if (!fp) {
-		php_printf("Could not open input file: %s\n", script_file);
+		fprintf(stderr, "Could not open input file: %s\n", script_file);
```

Comment:
This is quite important error, can one test asserting the stderr output (and no stdout output) be added?

Reply:
There were tests that checked for stdout, I modified them now such that they check stdout is empty and the error is in stderr.